doc/admin-guide/plugins/rate_limit.en.rst

```diff
@@ -84,26 +84,34 @@ This example shows a simple rate limiting of `128` concurrently active client
 transactions, with a maximum queue size of `256`. The default of HTTP status
 code `429` is used when queue is full.
 
+::
```

Comment:
I think you could do
```
when queue is full. ::
```
instead of putting it on a separate line. See [here](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#literal-blocks).

Reply:
Indeed that works and was my original version. Then @bneradt said
```
Good improvement.

As a small tweak, can we put the :: on a line by itself? That seems more common:

https://raw.githubusercontent.com/apache/trafficserver/master/doc/admin-guide/configuration/transparent-proxy/router-inline.en.rst
https://raw.githubusercontent.com/apache/trafficserver/master/doc/developer-guide/debugging/debug-builds.en.rst
To name a couple examples. Let me know if you had a reason to put the :: at the end of the previous line that I'm not aware of.```

 So I changed it to this version.

Reply:
Either way is fine with me. In the ATS docs I've worked with the `::` has been on a separate line, so I got used to that and thought that was our standard. If @SolidWallOfCode prefers to keep it at the end of the previous, then I'm fine with that. I notice we do both in our code.

Thanks for your patience with this @mlibbey.

Reply:
I suppose when Jon wrote
https://docs.trafficserver.apache.org/en/9.0.x/_sources/developer-guide/documentation/conventions.en.rst.txt
he used :: on the initial lines throughout. and in 
https://docs.trafficserver.apache.org/en/9.0.x/developer-guide/documentation/conventions.en.html#block-content
the text has examples like that `.. code::`; `.. sidebar::`, etc.

So, thinking we should go the same line route. If you agree Brian, will change back.

Reply:
Sounds good. Thanks!